nmrglue/util/xcpy.py

```diff
@@ -273,7 +273,7 @@ def run(cpython, script, pass_current_folder=True, use_shell=None, dry=None):
     args = [cpython, script] + cd
 
     if dry:
-        MSG("The following command will be executed: \n" + " ".join(args))
+        MSG("The following command will be executed:\n" + " ".join(args))
```

Comment:
I have removed the spurious space before `\n`. I am wondering whether the intent was to have `\n` after `" ".join(args)`.

Reply:
I'm not sure on the intent here.  Likely just a typo or overlooked extra character.